docs/PORTING.md

```diff
@@ -168,23 +181,39 @@ int     flash_area_id_from_multi_image_slot(int image_index, int slot);
 int     flash_area_id_to_multi_image_slot(int image_index, int area_id);
 ```
 
-**Note:** As of writing, it is possible that mcuboot will open a flash area multiple times simultaneously (through nested calls to `flash_area_open`). As a result, mcuboot may call `flash_area_close` on a flash area that is still opened by another part of mcuboot. As a workaround when porting, it may be necessary to implement a counter of the number of times a given flash area has been opened by mcuboot. The `flash_area_close` implementation should only fully deinitialize the underlying flash area when the open counter is decremented to 0. See [this GitHub PR](https://github.com/mcu-tools/mcuboot/pull/894/) for a more detailed discussion.
+---
+***Note***
+
+*Currently, MCUboot may open a flash area multiple times simultaneously*
+*(through nested calls to `flash_area_open`). As a result, MCUboot may call*
+*`flash_area_close` on a flash area that is still opened by another part of*
+*MCUboot. As a workaround when porting, it may be necessary to implement a*
+*counter of the number of times a given flash area has been opened by*
+*MCUboot. The `flash_area_close` implementation should only fully*
+*deinitialize the underlying flash area when the open counter is*
+*decremented to 0.*
+
+*See [this GitHub PR](https://github.com/mcu-tools/mcuboot/pull/894/) for*
+*more details.*
+
+---
 
-## Memory management for mbed TLS
+## Memory management for Mbed TLS
 
-`mbed TLS` employs dynamic allocation of memory, making use of the pair
-`calloc/free`. If `mbed TLS` is to be used for crypto, your target RTOS
-needs to provide this pair of function.
+`Mbed TLS` employs dynamic allocation of memory, making use of the pair
+`calloc/free`. If `Mbed TLS` is to be used for crypto, your target RTOS
+needs to provide this pair of functions.
 
-To configure the what functions are called when allocating/deallocating
-memory `mbed TLS` uses the following call:
+To configure which functions are called when allocating and deallocating
```

Comment:
I think that this should be: "To configure functions that", without comma after "memory".
The "which" introduces non-restrictive clause, but the sentence is "restrictive" because it says about specific functions: the functions that allocate and deallocate.

Reply:
Merriam Webster says this: https://www.merriam-webster.com/words-at-play/when-to-use-that-and-which "In today's usage which and that are both used to introduce restrictive clauses..."